DTDL/publication.json

```diff
@@ -0,0 +1,100 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:io:tributech:publication;1",
+  "@type": "Interface",
+  "displayName": "Publication",
```

Comment:
The actual content for some of the properties (e.g. InvitedNodes, Requests,..) will depend on the (auth) context in which they are retrieved. Owner vs. Consumer
Do you think we can/should express this with a single Publication twin or do we maybe need to distinguish between owning and consuming!?

This question will apply to multiple twins.

Reply:
i think it makes sense to separate them into different twins with a common base

Reply:
yes that sounds like a clean approach (might complicate usage at some points, but should be handle able.)